### ISSUE TITLE
Add utility function for handling missing values after reading SAS data

### ISSUE THREAD

**Issue by nanxstats** [Merck/metalite#158]
Our everyday workflow involves converting SAS data to R dataset using `haven::read_sas()`. However, in most of the cases, there is an additional step to convert SAS missing values for characters to `NA`s in R. This is not handled by the `read_sas()` call. It would be great to standardize this flow and reduce frictions by implementing a reusable utility function in metalite.

This issue was discussed in this [pharmaverse blog post](https://pharmaverse.github.io/blog/posts/2023-07-10_blanks_and_nas/blanks_and_nas.html). Theoretically, it can be handled by introducing admiral as a dependency. However, it currently has a non-trivial number of dependencies that we might want to minimize. So, implementing this in the zero-dependency metalite package could be ideal.

cc @julianschmocker @BrianLang

**Comment by elong0527** [comment]
I'd hesitate to guess user's intention. What if people intentionally use "" to represent a blank cell?

Should we either wait haven enable a proper behavior or write an example to guide people how to explicitly handle it?

e.g. 'mutate(across(is.character), function(x) ifelse(x == "", NA, x)'

**Comment by nanxstats** [comment]
I agree, the behavior should be made transparent and flexible, with a sensible default so that user land code is not tedious.

I would say instead of metalite, this could be a good fit for the "msdtools" package if that ever gets priority next year.

**Comment by elong0527** [comment]
Closing the issue as this helper function is more suitable for other packages.
